spec/index.spec.js

```diff
@@ -412,6 +415,85 @@ describe('axiosRetry(axios, { retries, retryDelay })', () => {
   });
 });
 
+describe('axiosRetry(axios, { retries, onRetry })', () => {
+  afterEach(() => {
+    nock.cleanAll();
+    nock.enableNetConnect();
+  });
+
+  describe('axiosRetry(axios, { retries, onRetry })', () => {
```

Comment:
I guess you don't need that describe (and afterEach) declarations, since they just duplicate the one above.

Reply:
@jmrapp1 is there any reason to duplicate this?